lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/verify/koarchitectureassert/KoArchitectureAssertTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.lemonappdev.konsist.core.verify.koarchitectureassert
+
+import com.lemonappdev.konsist.api.Konsist
+import com.lemonappdev.konsist.core.architecture.Layer
+import com.lemonappdev.konsist.core.exception.KoCheckFailedException
+import com.lemonappdev.konsist.core.verify.assert
+import org.amshove.kluent.shouldThrow
+import org.junit.jupiter.api.Test
+
```

Comment:
`KoArchitectureAssertTest` - I added this test class to `com/lemonappdev/konsist/core/verify` to keep the structure but it’s not an integration test. Should I have move it to `test` package?